storybook/stories/API/chart/FunnelChart.stories.tsx

```diff
@@ -17,6 +17,16 @@ export default {
         category: 'General',
       },
     },
+    shape: {
+      description: 'The customized shape to be rendered.',
+      table: {
+        type: {
+          summary: 'Function | boolean | ReactElement | object',
+        },
+        defaultValue: undefined,
+        category: 'General',
+      },
+    },
```

Comment:
I believe this is documented in multiple places. Should we extract a shared description, and import it? 

That would enforce standardised user facing communication for this property.